packages/actor-init-sparql/config/sets/sparql-queryoperators.json

```diff
@@ -109,6 +117,12 @@
       "cbqo:mediatorQueryOperation": { "@id": "config-sets:sparql-queryoperators.json#mediatorQueryOperation" }
     },
 
+    {
+      "@id": "config-sets:sparql-queryoperators.json#myFilterSparqleeQueryOperator",
```

Comment:
Oh, another thing. You can remove `ActorQueryOperationFilterDirect ` from here (also as a dependency)